ami/client/flowchart.py

```diff
@@ -66,7 +66,7 @@ def update_title(filename):
         title = 'AMI Client'
         if hutch:
             title += f' hutch: {hutch}'
-        if filename:
+        if filename is not None:
```

Comment:
This is unrelated, but something that I noticed randomly. If you click new and then save it will save a chart as just '.fc'.